### DISCUSSION TITLE
HTML Translation: Line Offset in case of New Lines addition.

### DISCUSSION THREAD

**Discussion by graebertshiva** [WeblateOrg/weblate#7023] [General]
Hi @nijel 

I have seen if we add some new Lines in between in HTML Base Language i.e. English, then the lines are offsetting in Other corresponding Language Files like German. 

is there anything i am missed to add some check like that so that i will be able to resolve this line offsetting. or do i need to add anything in my files like id?

**Comment by nijel** [comment]
Also asked at https://github.com/WeblateOrg/weblate/issues/7024
